tests/test_config.py

```diff
@@ -165,6 +165,23 @@ def test_log_config_yaml(
     mocked_logging_config_module.dictConfig.assert_called_once_with(logging_config)
 
 
+def test_log_config_yml(
```

Comment:
haven't looked at it too closely but it seems this is a full repeat of `test_log_config_yaml` so maybe a better option would be to use a 
`@pytest.mark.parametrize("config_filename", ["log_config.yml", "log_config.yaml"])`

Reply:
Thanks for the hint, @euri10, I was too lazy and indeed just copied the existing test. Now it's parametrized.